src/multi_select.coffee

```diff
@@ -1,4 +1,4 @@
-Ember.Widgets.MultiSelectOptionView = Ember.View.extend
```

Comment:
@cyril-sf You are right. But the original name is not very well-written. We have the SelectionOptionView in select component indicating the items in the dropdown while here it is the selected items (selected pills) which has a completely different meaning.
What should we do in this case?